controllers/scopeinstance_controller.go

```diff
@@ -364,3 +430,20 @@ func (r *ScopeInstanceReconciler) mapToScopeInstance(obj client.Object) (request
 
 	return
 }
+
+func (r *ScopeInstanceReconciler) updateScopeTemplateCondition(ctx context.Context, si *operatorsv1.ScopeInstance, condition metav1.Condition) {
+	// Get the latest instance of the ScopeTemplate in case it has been updated
+	tempSi := &operatorsv1.ScopeInstance{}
+	err := r.Client.Get(ctx, client.ObjectKeyFromObject(si), tempSi)
+	if err != nil {
+		log.Log.Error(err, "failed to get the latest version of the ScopeInstance to update the ScopeInstance status")
+		return
+	}
```

Comment:
Ditto here (same comment from other PR): https://github.com/awgreene/oria-operator/pull/20/files#r965243370

Reply:
Should be resolved in commit https://github.com/awgreene/oria-operator/pull/21/commits/6f670409b0304ae9d63cc1b7e30fab1c8b19e094

It is changed to return an error and everywhere it is used checks if the error is not `nil`. If it is not `nil` it requeues and returns an error.

Let me know how this looks!